include/zephyr/lorawan/lorawan.h

```diff
@@ -295,6 +295,16 @@ int lorawan_set_conf_msg_tries(uint8_t tries);
  */
 void lorawan_enable_adr(bool enable);
 
+/**
+ * @brief Set the channels mask.
+ *
```

Comment:
It would be good to add some notes here about when you would want to use this function

Reply:
Fixed.